stamen/map-compare
# Add BASE_PATH option on build

Similar to the work done in [mapbox-gl-style-diff-viewer](https://github.com/stamen/mapbox-gl-style-diff-viewer/), this adds an option to the build script to change the base path of the app. Since we will likely be deploying this relative to some path (eg, `/tools/compare/`) this will help with deploy.

To test:
 1. `BASE_PATH=/my-compare-tool/ yarn build`
 2. `cp -r public my-compare-tool` (simulate copying to server at that path)
 3. `python3 -m http.server` (simulate running server at the root of the project)
 4. Open `http://localhost:8000/my-compare-tool/` in a browser, tool should work as expected.
